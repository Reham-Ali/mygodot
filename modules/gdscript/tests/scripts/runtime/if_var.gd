func foo(x: int):
	return x > 0

func test():
	if var x = 25:
		print(x)

	if var x = 25:
		print(x)
	else:
		print("ff")

	if var x = foo(0):
		print(x)

	if var x = foo(0):
		print(x)
	else:
		print("fff")

	if var x := signi(100):
		print(x)
