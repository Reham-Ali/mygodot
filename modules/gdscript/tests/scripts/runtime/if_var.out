GDTEST_OK
25
25
fff
1
